scripts/mbedtls_framework/code_wrapper/psa_wrapper.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python3
+"""Generate wrapper functions for PSA function calls.
+"""
+
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+import argparse
+import itertools
+import os
+from typing import Any, Iterator, List, Dict, Collection, Optional, Tuple
+
+from mbedtls_framework import build_tree
+from mbedtls_framework import c_parsing_helper
+from mbedtls_framework import c_wrapper_generator
+from mbedtls_framework import typing_util
+
+from mbedtls_framework.code_wrapper.psa_buffer import BufferParameter
+from textwrap import dedent
```

Comment:
Please group standard library imports, per [PEP 8](https://peps.python.org/pep-0008/#imports). (Pylint will nag you anyway, once you run it.) Also, PEP 8 doesn't require it, but please keep imports in the same group in module alphabetical order, that makes it easier to find something by scanning a list, so `textwrap` goes between `os` and `typing`.